src/modules/admin/components/users/UserAuditPage.tsx

```diff
@@ -73,64 +116,86 @@ const UserAuditPage: React.FC<Props> = ({ entityType }) => {
 
   return (
     <>
-      <PageTitle title={`Access History for ${user.name}`} />
+      <PageTitle title={`${user.name} Audit History`} />
+      <Alert severity='warning' sx={{ mb: 2 }}>
+        Data on this page may be delayed by up to an hour. Recent user activity
+        may not be immediately visible.
+      </Alert>
```

Comment:
This is only true for the User Access history, since it requires some periodic processing of activity logs. The User Edits show up right away. So can we move this alert down into the card?